Code/Framework/AzCore/AzCore/Memory/Memory.h

```diff
@@ -838,17 +472,17 @@ namespace AZ
             : m_allocator(rhs.m_allocator)
             , m_name(name) { (void)rhs; }
         AZ_FORCE_INLINE AZStdIAllocator& operator=(const AZStdIAllocator& rhs) { m_allocator = rhs.m_allocator; m_name = rhs.m_name; return *this; }
-        AZ_FORCE_INLINE pointer_type allocate(size_t byteSize, size_t alignment, int flags = 0)
+        AZ_FORCE_INLINE pointer allocate(size_t byteSize, size_t alignment)
```

Comment:
Isn't AZStdIAllocator the same as the AllocatorPointer wrapper in AllocatorWrapper.h?